app/views/shared/_one_time_code_input.html.erb

```diff
@@ -15,22 +15,23 @@ locals:
    classes = ['field monospace usa-input one-time-code-input']
    classes << local_assigns.delete(:class) if local_assigns[:class] %>
 
-
-<%= text_field_tag(
-      name,
-      nil,
-      'aria-invalid': 'false',
-      'data-transport': transport,
-      pattern: numeric ? '[0-9]*' : '[a-zA-Z0-9]*',
-      maxlength: TwoFactorAuthenticatable::DIRECT_OTP_LENGTH,
-      autocomplete: 'one-time-code',
-      inputmode: numeric ? 'numeric' : 'text',
-      type: 'text',
-      **local_assigns,
-      class: classes,
-    ) %>
-<span class='usa-error-message display-if-invalid display-if-invalid--value-missing margin-top-1' role='alert'>
-  <%= t('simple_form.required.text') %>
-</span>
+<div>
```

Comment:
The reason for this is that process list styling will add a margin to the element immediately following the heading. The script associated with this partial will dynamically add a hidden input field before the text input. I could have solved this by adding a wrapper either here or in the content of the process list item. Viewing this more as a component, I think it makes sense that there would be one root-level element, and any dynamic element behavior would occur within the container of the component, and not as a sibling of one of multiple top-level elements.